server/flyway/local_sql/V1001__add_bceid_delegated_admin_for_testing.sql

```diff
@@ -0,0 +1,71 @@
+-- create user for running test, test for the security check for delegated admin grant/remove access
+
+-- create two business bceid users from the same organization, one from different organization
+INSERT INTO app_fam.fam_user (
+    user_name,
+    user_type_code,
+    cognito_user_id,
+    business_guid,
+    create_user
+)
+VALUES
+('TEST-3-LOAD-CHILD-1','B','test-bceidbusiness_bda2a1e212244dc2b9f9522057mangledc58bbb@bceidbusiness', 'E7C0431DA55D4ACA9FA901EE2C91CB3B', CURRENT_USER),
+('LOAD-3-TEST', 'B', 'test-bceidbusiness_532905de0aa24923ae535428mangledf171bf13@bceidbusiness', 'E7C0431DA55D4ACA9FA901EE2C91CB3B', CURRENT_USER),
```

Comment:
After we enable the storing of business_guid when creating the user in the future, we won't need to create "LOAD-3-TEST" and "LOAD-4-TEST" in this file anymore. Currently we need to test the business bceid users from same vs different organization, we will need the business_guid, so have to add them here.